src/symbolic/ellipsoidal_transitions.jl

```diff
@@ -314,13 +313,13 @@ function _compute_base_cell(r::SVector{S}) where {S}
     return polyhedron(intersect(baseCellList...))
 end
 
-"""
-    _provide_P(subsys::HybridSystems.ConstrainedAffineControlDiscreteSystem, optimizer)
+#
```

Comment:
Why is this commented?

Reply:
Because we agreed that this should not necessarily go to the doc for the moment, but if I do not comment these lines, there is an error because there is "documentation text" which is not in the documentation.